app/models/megalith.rb

```diff
@@ -1,2 +1,9 @@
 class Megalith < ApplicationRecord
+  has_many :ratings
+  has_many :megalith_photos
+  has_many :favourites
```

Comment:
not sure this needs to be here? Will we need ot call lith.favourites somewhere? I think only brian.favourites